packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1834,6 +1835,9 @@ class TimePickerDialog extends StatefulWidget {
   /// Optionally provide your own minute label text.
   final String? minuteLabelText;
 
+  /// If set minutes selection is active when the dialog is shown.
+  final bool focusOnMinutes;
```

Comment:
Maybe this should use _TimePickerMode? (a public version of it, that is). That way it would be easier to understand what the values mean.